db/c.cc

```diff
@@ -6592,4 +6643,59 @@ void rocksdb_enable_manual_compaction(rocksdb_t* db) {
   db->rep->EnableManualCompaction();
 }
 
+rocksdb_statistics_histogram_data_t*
+rocksdb_statistics_histogram_data_create() {
+  return new rocksdb_statistics_histogram_data_t;
+}
```

Comment:
See my reply to your discussion - I do agree your previous `return new rocksdb_statistics_histogram_data_t{};` is better!